internal/server/search.go

```diff
@@ -239,17 +240,19 @@ func (server *Server) cmdSearchArgs(
 		}
 		// radius is optional for nearby, but mandatory for others
 		if cmd == "nearby" {
-			if vs, smeters, ok = tokenval(vs); ok && smeters != "" {
-				if meters, err = strconv.ParseFloat(smeters, 64); err != nil {
-					err = errInvalidArgument(smeters)
-					return
-				}
-				if meters < 0 {
-					err = errInvalidArgument(smeters)
-					return
+			meters = -1
+
+			if len(vs) > 0 && strings.ToLower(vs[0]) != "clipby" {
+				if vs, smeters, ok = tokenval(vs); ok && smeters != "" {
+					if meters, err = strconv.ParseFloat(smeters, 64); err != nil {
+						err = errInvalidArgument(smeters)
+						return
+					}
+					if meters < 0 {
+						err = errInvalidArgument(smeters)
+						return
+					}
```

Comment:
This change doesn't seem necessary as I understand the code: the object should be the last thing in the command, so clipby should never come after it